theme/navigateregions.less

```diff
@@ -0,0 +1,72 @@
+/* Import all components */
+@import 'themes/default/globals/site.variables';
+@import 'themes/pxt/globals/site.variables';
+
+/* Reference import */
+@import (reference) "semantic.less";
+
+.navigate-regions-container {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0,0,0,0);
+    width: 100%;
+    height: 100%;
+    z-index: @modalDimmerZIndex;
+
+    .region-button {
+        position: absolute;
+        background-color: rgba(0, 0, 0, .6);
+        border-radius: 0;
+        border: 3px solid white;
```

Comment:
I realize it'll be a little backwards (using foreground theme colors for a background and vice versa) but it's probably still slightly preferable to use neutral theme values in this file instead of `white` and `back`.  I'd recommend trying `pxt-neutral-background1` instead of white and `pxt-neutral-foreground1` instead of black.

Reply:
pxt-neutral-background1 (for light theme white) on a background of --pxt-neutral-alpha80 in high contrast seems to be black on black.

I've tweaked those cases to use --pxt-secondary-foreground for what was white. Hope that's OK.

Found an amazing consequence of the high contrast theme setting `z-index: 1` on button hover. Hovering over the editor region made the mini-sim vanish. Tweaked z-index as a result.

Reply:
Thanks for double checking that. I did not think to check high-contrast. Your notes all seem good to me!